P: Openload #259

zoreu opened the pull request:
fix video download

krystian3w commented:
No worth, only repair SPINNET (remove bad)?

` @@||oload.vip^$generichide ` should be moved to: `Adblock Warning Removal List`

zoreu commented:
@krystian3w I removed the: `abort-current-inline-script document.getElementById isSandboxed` 
and added @@||oload.vip^$generichide, not everyone has Adblock Warning Removal List
